shanoir-ng-datasets/src/main/java/org/shanoir/ng/dicom/DicomProcessing.java

```diff
@@ -26,18 +26,18 @@
 @Service
 public class DicomProcessing {
 	
-	public Attributes getDicomObjectAttributes(DatasetFile image, boolean isEnhancedMR) throws IOException {
+	public Attributes getDicomObjectAttributes(DatasetFile image, Boolean isEnhancedMR) throws IOException {
 		File dicomFile = new File(image.getPath());
 		try (DicomInputStream dIS = new DicomInputStream(dicomFile)) {
 			Attributes datasetAttributes;
-			if (isEnhancedMR) {
+			if (isEnhancedMR != null && isEnhancedMR) {
 				// In case of Enhanced MR, we need to the pixel data in order to use Dcm4chee emf extract method.
-				datasetAttributes = dIS.readDataset(-1, -1);
+				datasetAttributes = dIS.readDataset(-1);
 			} else {
 				// Else we do not load the picture in Ram for faster performance.
-				datasetAttributes = dIS.readDataset(-1, Tag.PixelData);
+				datasetAttributes = dIS.readDataset(Tag.PixelData);
 			}
-			
+			dIS.close();
```

Comment:
This is not necessary with "try with ressource" as done:
https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html